scipy/interpolate/_polyint.py

```diff
@@ -677,8 +743,91 @@ def _evaluate(self, x):
                 p[r[:-1]] = self.yi[r[-1]]
         return p
 
+    def derivative(self, x, der=1):
+        """
+        Evaluate one derivative of the polynomial at the point x
 
-def barycentric_interpolate(xi, yi, x, axis=0):
+        Parameters
+        ----------
+        x : array_like
+            Point or points at which to evaluate the derivatives
+        der : integer, optional
+            Which derivative to extract. This number includes the
+            function value as 0th derivative.
+
+        Returns
+        -------
+        d : ndarray
+            Derivative interpolated at the x-points. Shape of d is
+            determined by replacing the interpolation axis in the
+            original array with the shape of x.
```

Comment:
```suggestion
            original array with the shape of ``x``.
```

Reply:
We use single quotes for parameters of the given function. We only use double quotes for things like one liner code examples or some simple maths sometimes.